pkg/scheduler/framework/plugins/volumebinding/OWNERS

```diff
@@ -1,6 +1,11 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
+approvers:
```

Comment:
@Huang-Wei what are your thoughts on this?

Another option is to create folder within `plugins/volumebinding` which is completely owned by sig storage, but the parent folder is still owned by sig-scheduling. But perhaps that's overkill.

Reply:
It is still not clear for me. We need remove this file to return ownership to sig-scheduling or we need to do something like this:
```
approvers:
- sig-scheduling-maintainers
reviewers:
- sig-scheduling
- cofyc
- msau42
- lichuqiang
```
(leave original reviewers 
https://github.com/kubernetes/kubernetes/blob/9ff3b7e744b34c099c1405d9add192adbef0b6b1/pkg/controller/volume/scheduling/OWNERS)

WDYT @Huang-Wei ?

Reply:
You can leave it like this